Companies.py

```diff
@@ -0,0 +1,9 @@
+# Defining 2 Companies Eleks and Toshiba
+Eleks = ['Albina', 'Kostiantyn', 'Bohdan', 'James', 'Sam', 'Peter']
+Toshiba = ['Halyna', 'Oksana', 'Paul', 'Peter', 'Soul', 'Bohdan', 'Katalin']
+
+# Company Toshiba takes over Eleks company
+Toshiba.extend(Eleks)
```

Comment:
it is not completed)) after this manipulation eleks still has employees. Please add one line of code to fix it